app.js

```diff
@@ -0,0 +1,40 @@
+const express = require("express");
+const {
+  getCategories
+} = require("./controllers/bg.controller");
+
+const app = express()
+
+app.use(express.json());
+
+app.get("/api/categories", getCategories);
+
+
+app.use((err, req, res, next) => {
```

Comment:
The 404 and 500 error handlers can are good as these have been tested/can be useful at this point to have, but please remove the rest as they are not necessary for this task and have not been tested for.